Makefile

```diff
@@ -15,17 +15,18 @@ bats-tests: ## Run Helm chart bats tests.
 # ===========> Control Plane Targets
 
 control-plane-dev: ## Build consul-k8s-control-plane binary.
-	@$(SHELL) $(CURDIR)/control-plane/build-support/scripts/build-local.sh -o $(GOOS) -a $(GOARCH)
+	@$(SHELL) $(CURDIR)/control-plane/build-support/scripts/build-local.sh -o linux -a amd64
```

Comment:
Is this temporary? I figure you made the change to make debugging easier?

Reply:
It is temporary as there are changes in main that will need to come over here for the Makefile/Dockerfile.